package.json

```diff
@@ -170,6 +170,7 @@
     "source-map": "^0.7.3",
     "style-loader": "^0.23.1",
     "superagent": "^5.2.2",
+    "telegraf": "^4.3.0",
```

Comment:
why do you add `telegraf`?

Reply:
Please regenerate the `yarn.lock` file by running `yarn` to remove this dependency there as well.